fairtools/MCConfigurator/FairYamlVMCConfig.h

```diff
@@ -16,6 +16,11 @@
 #include "FairGenericVMCConfig.h"
 
 #include <string>
+
+// prevent yaml-cpp/node/detail/iterator.h:48:54:
+//  error: no member named 'next' in namespace 'boost'
+#include <boost/next_prior.hpp>
```

Comment:
Could it be that you are using yaml-cpp < 0.6? I guess, we only support 0.6+, which does not depend on Boost any more.

Reply:
This looks a lot like #1199?